editoast/src/modelsv2/prelude/list.rs

```diff
@@ -158,6 +158,18 @@ impl<M: Model + 'static> SelectionSettings<M> {
         self.paginate_counting = paginate;
         self
     }
+
+    /// Returns a tuple (page_number, page_size) based on the provided
+    /// [SelectionSettings::limit] and [SelectionSettings::offset] settings
+    ///
+    /// The page number is 1-based.
```

Comment:
The documentation mentions `page_number` and `page_size` but the code mentions `offset` and `limit`. Maybe one or the other should be renamed to match.

Reply:
This is intentional, because these terms are used in different contexts:

* `page_number` and `page_size` are used in the views to represent pagination settings
* `offset` and `limit` are used in the models to represent a range of rows in the DB for listings (these values _could_ be derived from pagination settings, _or not_)

Besides `limit == page_size`, but `page_number != offset`. Wdyt?

Reply:
Your explanation helps a lot... but it was not obvious from the comment that there really were 2 different concepts. So the question becomes how do we improve the documentation maybe?

Reply:
Added more documentation in https://github.com/OpenRailAssociation/osrd/pull/7397/commits/634c77c4b5588c7784b2a55d0f43f0ebe98e4718.